gradle.properties

```diff
@@ -7,6 +7,9 @@
 # For more details on how to configure your build environment visit
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 
+android.enableJetifier=true
```

Comment:
Is this necessary after getting off jetifier?  Thanks for doing all these updates.

Reply:
That's a great question and one I was wondering myself. So this was added automatically by the AndroidX migration tool and my thinking was that it was only applicable when building the sample project. The only external dependency there is the State Saver library and I've testing things out a bit and it turns out that version 1.3.1 of that library requires the support libraries and therefore Jetifier is required but after I update to 1.4.1 it is not. So I'm going to disable Jetifier in the commit where I update that dependency. This likely won't have any effect on the library itself but its still probably a good idea to disable it for the sample library if we can. Thanks for pointing this out!